Move COmplete Vehicles Section to top level

hamishwillee opened the pull request:
As discussed @bkueng

bkueng commented on en/complete_vehicles/README.md:
```diff
@@ -0,0 +1,14 @@
+  * [Crazyflie 2.0](../complete_vehicles/crazyflie2.md)
+  * [Parrot Bebop](../complete_vehicles/bebop.md)
+  * [BetaFPV Beta75X 2S Brushless Whoop](../complete_vehicles/betafpv_beta75x.md)
+  * Teal One
```
This runs natively PX4

bkueng commented on en/complete_vehicles/README.md:
```diff
@@ -0,0 +1,14 @@
+  * [BetaFPV Beta75X 2S Brushless Whoop](../complete_vehicles/betafpv_beta75x.md)
+  * Teal One
+- Whole-vehicle hardware reference platforms that use PX4:
+  * [Intel® Aero Ready to Fly Drone](../complete_vehicles/intel_aero.md) - UAV development platform with high-end autopilot and integrated onboard computer.
```
I'd say it's not that 'high-end' anymore

bkueng commented on en/getting_started/flight_controller_selection.md:
```diff
@@ -20,22 +21,20 @@ Controller | Description
 ## Commercial UAVs that can run PX4
 
-PX4 has been ported to a number of popular commercial drone products, as listed below. This allows you to add mission planning and other PX Flight modes to your vehicle. 
+PX4 is available on many popular commercial drone products, including some that ship with PX4 and others that can be updated with PX4 (allowing you to add mission planning and other PX Flight modes to your vehicle).
```
typo: PX

hamishwillee commented:
Hi @bkueng . Thanks for the review! I fixed your feedback and added new list of vehicles that support PX4 natively in https://github.com/PX4/px4_user_guide/pull/416/commits/93e9b5063b3acd1715a74a276daa1ea68115ea54#diff-708de220e4858a7cc55b77ed0e473e05R8 - can you please sanity check that.

Already merged.

bkueng commented:
So I take it you did not want to make the distinction between upstream supported and PX4-based vehicles?

bkueng commented:
> I thought I had - what do you mean by upstream supported? Aren't they Consumer drones that can be updated to run PX4?

No. Let me try to explain with an example: The H520 is based on PX4 (meaning Yuneec uses the PX4 Firmware/source code), and adds a separate board configuration and some other code customizations. These are not public, but they are required to run PX4 on the H520 and thus it is impossible to flash/use upstream PX4 on that vehicle.

hamishwillee commented:
Thanks @bkueng . Yes it makes sense to separate those out because they are different - ie use customised PX4 from another vendor. What is the set of these? H520 - the other Yuneec's? what else?

bkueng commented:
Yes it includes all of the Yuneec vehicles.
The ones from the Consumer list that are actually upstream supported are only these:
    * [Teal One](https://tealdrones.com/teal-one/)
    * [Vertical Technologies DeltaQuad](https://px4.io/portfolio/deltaquad-vtol/)

hamishwillee commented:
@bkueng Just came back to this. So is this how the list would look (I found the last comment above confusing).

- Consumer drones run PX4 natively/"out of the box":
  * Multicopter
    * [AeroSense Aerobo (AS-MC02-P)](https://px4.io/portfolio/aerosense-aerobo/)
    * [Airlango Mystic](https://px4.io/portfolio/airlango-mystic/)
    * [Airdog II](https://px4.io/portfolio/airdog-ii/)
  * Fixed Wing:
    * [Sentera PXH](https://px4.io/portfolio/sentera-phx/)
  * VTOL
    * [Flightwave Edge](https://px4.io/portfolio/flywave-edge/)
    * [AeroSense VTOL](https://px4.io/portfolio/sony-aerosense/)
    * [WingtraOne Tailsitter VTOL](https://px4.io/portfolio/wingtraone-tailsitter-vtol/)
- Consumer drones that can be updated to run PX4 (from other flight stacks):
  * [Crazyflie 2.0](../complete_vehicles/crazyflie2.md)
  * [Parrot Bebop](../complete_vehicles/bebop.md)
  * [BetaFPV Beta75X 2S Brushless Whoop](../complete_vehicles/betafpv_beta75x.md)
- Whole-vehicle hardware reference platforms that use PX4:
  * [Intel® Aero Ready to Fly Drone](../complete_vehicles/intel_aero.md) - UAV development platform with autopilot and integrated onboard computer.
- Consumer drones run a custom version of PX4 (supported by their vendors):
  * Multicopter
    * [Teal One](https://tealdrones.com/teal-one/)
    * [Yuneec Typhoon H Plus](https://px4.io/portfolio/yuneec-typhoon-h-plus/)
    * [Yuneec Mantis Q](https://px4.io/portfolio/yuneec-mantis-q/)
    * [Yuneec H520](https://px4.io/portfolio/yuneec-h520-hexacopter/)
  * VTOL
    * [Vertical Technologies DeltaQuad](https://px4.io/portfolio/deltaquad-vtol/)

Antiheavy commented:
@hamishwillee.  Sentera PHX fixed wing falls under the “custom version of px4” category.  Thanks.

bkueng commented:
@hamishwillee this is how it should be (some of them I'm not quite sure, most likely the fall into custom):
- Consumer drones run PX4 natively/"out of the box":
  * Multicopter
    * [Teal One](https://tealdrones.com/teal-one/)
  * VTOL
    * [Vertical Technologies DeltaQuad](https://px4.io/portfolio/deltaquad-vtol/)
- Consumer drones that can be updated to run PX4 (from other flight stacks):
  * [Crazyflie 2.0](../complete_vehicles/crazyflie2.md)
  * [Parrot Bebop](../complete_vehicles/bebop.md)
  * [BetaFPV Beta75X 2S Brushless Whoop](../complete_vehicles/betafpv_beta75x.md)
- Whole-vehicle hardware reference platforms that use PX4:
  * [Intel® Aero Ready to Fly Drone](../complete_vehicles/intel_aero.md) - UAV development platform with autopilot and integrated onboard computer.
- Consumer drones run a custom version of PX4 (supported by their vendors):
  * Multicopter
    * [Yuneec Typhoon H Plus](https://px4.io/portfolio/yuneec-typhoon-h-plus/)
    * [Yuneec Mantis Q](https://px4.io/portfolio/yuneec-mantis-q/)
    * [Yuneec H520](https://px4.io/portfolio/yuneec-h520-hexacopter/)
    * [Airlango Mystic](https://px4.io/portfolio/airlango-mystic/)
    * [Airdog II](https://px4.io/portfolio/airdog-ii/)
    * [AeroSense Aerobo (AS-MC02-P)](https://px4.io/portfolio/aerosense-aerobo/) *I don't know*
  * Fixed Wing:
    * [Sentera PXH](https://px4.io/portfolio/sentera-phx/)
  * VTOL
    * [Flightwave Edge](https://px4.io/portfolio/flywave-edge/)  *I don't know*
    * [AeroSense VTOL](https://px4.io/portfolio/sony-aerosense/)  *I don't know, and the link is dead*
    * [WingtraOne Tailsitter VTOL](https://px4.io/portfolio/wingtraone-tailsitter-vtol/)

hamishwillee commented:
Thanks @bkueng (and @Antiheavy ). I've updated as indicated. Removed the item with broken link - couldn't find a good replacement and there isn't much "realness" about it. I left the "I don't know" ones in place - better for users not to expect they can update them directly than to expect they can and not be able to.
